accord-core/src/main/java/accord/local/SafeCommandStore.java

```diff
@@ -66,7 +70,37 @@
         EXECUTES_AFTER
     }
     enum TestDep { WITH, WITHOUT, ANY_DEPS }
-    enum TestKind { Ws, RorWs }
+    enum TestKind implements Predicate<Txn.Kind>
```

Comment:
So with this do sync points only conflict in one direction and basically not block other transactions? Nice.